docs/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-last_review_date: "2025-02-08"
+last_review_date: "2025-06-16"
```

Comment:
Have you reviewed this whole document?

Reply:
@botantony Please just put this date back to what it was before unless you have.